src/astro/databases/google/bigquery.py

```diff
@@ -156,15 +192,14 @@ def merge_table(
             ON {' AND '.join(['T.' + col + '= S.' + col for col in target_conflict_columns])}\
             WHEN NOT MATCHED BY TARGET THEN INSERT ({','.join(target_columns)}) VALUES ({','.join(source_columns)})"
 
+        update_statement_map = ", ".join(
```

Comment:
@sunank200 Why do we have this change in this PR?

Reply:
Deepsource was failing. This solves this deepsource